hyperspy/io.py

```diff
@@ -52,6 +52,7 @@ def load(filenames=None,
          new_axis_name="stack_element",
          lazy=False,
          convert_units=False,
+         preferred_format=None,
```

Comment:
as above I think "hdf_format" might be more explicit and therefore better? Is there any issue with that?

Reply:
Ok, having read more of the code I see that the user could specify any format and try to use that reader to load some file. Given that, I think it might be best to go with just "format"?

In any case - don't forget to add a description to the Parameters section of this docstring.

Reply:
changed to file_format - note this impact usid as well...